docs/content/features/credentials_collectors/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Credentials Collectors"
+chapter: true
+pre: "<i class='fas fa-key'></i> "
+---
+
+# Credentials Collectors
+
+The Infection Monkey has multiple ways to steal credentials from compromised
+machines:
```

Comment:
We should provide more context about _why_ you would want to steal credentials and what Monkey can do with stolen credentials. How are stole credentials used?

Reply:
WDYT?
![image](https://github.com/user-attachments/assets/766f6524-7a8f-4cd1-9e8e-851c0f111ebc)

Reply:
What do you think about this format instead? 
![image](https://github.com/user-attachments/assets/bb7e2bed-661d-45b3-a1cf-adc9926cdfcc)

Reply:
We can do this with an empty h2 ("## ").
![image](https://github.com/user-attachments/assets/33b10b4e-6157-4689-babd-b2179ce9aea6)